recipes/vcf-validator/build.sh

```
@@ -1,11 +1,16 @@
-mkdir -p ${PREFIX}/bin
-# Check if linux or osx
+# Install additional dependencies
 if [ -z ${OSX_ARCH+x} ]; then
-  mv vcf_validator_linux ${PREFIX}/bin/vcf_validator
-  mv vcf_assembly_checker_linux ${PREFIX}/bin/vcf_assembly_checker
+  ./install_dependencies.sh linux
+  mkdir build
+  cd build
```

Shellcheck

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)